lms/templates/forgot_password_modal.html

```diff
@@ -49,10 +51,17 @@ <h2>${_("Password Reset")}</h2>
   (function() {
     $(document).delegate('#pwd_reset_form', 'ajax:success', function(data, json, xhr) {
       if(json.success) {
-        $("#password-reset").html(json.value);
+        $("#password-reset").text(json.value);
       } else {
         if($('#pwd_error').length == 0) {
-          $('#pwd_reset_form').prepend('<div id="pwd_error" class="modal-form-error">${_("Email is incorrect.")}</div>');
+        var msg = "${_("Email is incorrect.") | n, js_escaped_string}";
```

Comment:
This doesn't look indented correctly for the if.

Reply:
Fixed